src/Components/SignatureSideBar/Components/ClassApplyBtn.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Button, ButtonColor } from '@class101/ui';
+import styled from "styled-components";
+
+function ClassApplyBtn() {
+  return (
+    <ClassApplyBtnContainer>
+      <Button color={ButtonColor.RED} ><spna>클래스 신청하기</spna></Button>
```

Comment:
```suggestion
      <Button color={ButtonColor.RED} ><span>클래스 신청하기</span></Button>
```
1. 오타 수정해주세요
2. self closing 태그 써주세요

Reply:
버튼 안에  "클래스 신청하기" 를 넣어하는데 self closing 하면 버튼이랑 텍스트가 따로 놀아요 ㅠㅠ 
self closing 사용하면서 버튼 안에 텍스트를 넣을 수 있는 방법이 있을까요?